pkg/config/config.go

```diff
@@ -301,12 +301,20 @@ func (c *Config) GetDatabaseParams() dbparams.Database {
 }
 
 func (c *Config) GetKVParams() kvparams.KV {
-	return kvparams.KV{
+	p := kvparams.KV{
 		Type: c.values.Database.Type,
-		Postgres: &kvparams.Postgres{
-			ConnectionString: c.values.Database.ConnectionString.SecureValue(),
-		},
-		DynamoDB: &kvparams.DynamoDB{
+	}
+	switch {
+	case c.values.Database.Postgres != nil:
+		p.Postgres = &kvparams.Postgres{
+			ConnectionString:      c.values.Database.Postgres.ConnectionString.SecureValue(),
+			MaxIdleConnections:    c.values.Database.Postgres.MaxIdleConnections,
+			MaxOpenConnections:    c.values.Database.Postgres.MaxOpenConnections,
+			ConnectionMaxLifetime: c.values.Database.Postgres.ConnectionMaxLifetime,
```

Comment:
Do you wish to drop support for the old values as well and make it a breaking change? Because most users will continue to use them

Reply:
I don't see a way we keep them in this structure. Once we decided to create sections per driver implementation - it makes no sense to keep these on the root level. I suggest this will be added to the migration procedure. WDYT?

Reply:
I see a way around it. you can do something like:
```suggestion
	connectionStr := c.values.Database.Postgres.ConnectionString.SecureValue()
	if len(connectionStr) == 0 && len(c.values.Database.DeprecatedConnectionString.SecureValue()) == 0 {
		connectionStr = c.values.Database.DeprecatedConnectionString.SecureValue()
		log.Warning("Using deprecated connection string, use DATABASE_POSTGRES_CONNECTION_STRING instead. Support will be dropped soon.")
	}
	p.Postgres.ConnectionString = connectionStr
```